.github/workflows/rust.yml

```diff
@@ -20,17 +20,20 @@ jobs:
       matrix:
         runs-on: [ubuntu-22.04, windows-2022, macos-13]
         toolchain:
-          - "1.62"
+          - 1.62
```

Comment:
This should be quoted so yaml doesn't parse it as a float

Reply:
It's treated as a string by the parser either way?